pallets/ethereum-transaction/src/tests.rs

```diff
@@ -0,0 +1,131 @@
+use cfg_traits::ethereum::EthereumTransactor;
+use frame_support::{assert_ok, traits::fungible::Mutate};
+use pallet_evm::{AddressMapping, Error::BalanceLow};
+use sp_core::{crypto::AccountId32, H160, U256};
+use sp_runtime::DispatchError;
+
+use super::mock::*;
+use crate::pallet::Nonce;
+
+mod utils {
```

Comment:
Nit: Why don't we include this into the mock? That's the default substrate style.

Reply:
As in the `mock` module? We will only use it here as far as I can tell, why have it somewhere else?

Reply:
Thanks for the comment @lemunozm! Let's leave this as is.